helpers/ea_dir2cell.m

```diff
@@ -0,0 +1,9 @@
+function dicell=ea_dir2cell(di)
+dicell={};
+cnt=1;
+for d=1:length(di)
+    if ~strcmp(di(d).name(1),'.')
+        dicell{cnt}=di(d).name;
+        cnt=cnt+1;
+    end
+end
```

Comment:
1. Instead of input `di`, can think about input the folder name and do `dir` inside of the function.
2. Instead of for loop here, can use `dicell = {di.name}; dicell = dicell(~startsWith(dicell, '.'))`.
3. but, this will also exclude those files/folders which start with `.`, for example `.pp`
4. Can also simply use `ea_regexpdir(folder, '.*', 0, 'a')` to list all the files and folders under `folder`. Or if you just need to list all the folders, can use `ea_regexpdir(folder, '.*', 0, 'd')`

Reply:
Thanks! I will work on this and re commit